mimesis/utils.py

```diff
@@ -7,30 +7,35 @@
 import urllib.request as request
 from random import choice
 
-from mimesis.exceptions import UnexpectedGender, UnsupportedLocale
-from mimesis.settings import SUPPORTED_LOCALES
+from mimesis.exceptions import (
+    UnexpectedGender,
+    UnsupportedLocale,
+)
+import mimesis.settings as settings
+import mimesis.type as types
```

Comment:
I have collected all types in this module and import ones from it. Is it good idea?

Reply:
Yeap, it is!